apps/ui/src/components/providers/ThemeProvider.tsx

```diff
@@ -24,17 +24,30 @@ const pcglTheme: ThemeConfig = {
 	token: {
 		fontFamily:
 			"Open Sans, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, 'Helvetica Neue', Arial, 'Noto Sans', sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji'",
+
 		colorPrimary: '#C41D7F',
-		colorBgContainer: '#FFF0F6',
+		colorFillAlter: '#FFFFFF',
```

Comment:
#FFFFFF is just white and this is a common enough color that I don't think we need to redefine it several times for different components. Wondering if there's a way we can do something akin to `colorFillAlter: pcglTheme.color.white`

Reply:
Reviewed a little further, I'm more wondering about this name `colorFillAlter`; is it specific to Ant Design, or are we just adding a new name for white?

In StatusBannerWrapper we pass `backgroundColor: token.colorFillAlter`, but since we're just setting the `backgroundColor` property to the `colorFillAlter` value, we could probably substitute `token.colorFillAlter` for `token.color.white`

Reply:
Big fan of this suggestion. Refactoring themes

Reply:
OK, I do see the names have meaning in Ant so maybe it's just friction with their approach. But when we're just setting CSS properties like `backgroundColor` then adding standard color definitions can help keep code DRY and easily readable.

Reply:
So we are locked into properties set by the ThemeConfig provided by Ant design (like you suggested above), I have created a new object with our style guide colors (ones specifically that have been used in the docs) and using that object we can maintain DRY. Just pushed the changes I was thinking of initially, lmk how it is.